fleet_vehicle_inspection/models/fleet_vehicle_inspection.py

```diff
@@ -0,0 +1,196 @@
+# Copyright 2020 - TODAY, Marcel Savegnago - Escodoo https://www.escodoo.com.br
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class FleetVehicleInspection(models.Model):
+
+    _name = "fleet.vehicle.inspection"
+    _description = "Fleet Vehicle Inspection"
+    _inherit = ["mail.thread", "mail.activity.mixin"]
+
+    READONLY_STATES = {
+        "confirmed": [("readonly", True)],
+        "cancel": [("readonly", True)],
+    }
+
+    name = fields.Char(
+        "Reference", required=True, index=True, copy=False, default="New"
+    )
+
+    state = fields.Selection(
+        [("draft", "Draft"), ("confirmed", "Confirmed"), ("cancel", "Cancelled")],
+        string="Status",
+        copy=False,
+        index=True,
+        readonly=True,
+        track_visibility="onchange",
+        default="draft",
+        help=" * Draft: not confirmed yet.\n"
+        " * Confirmed: inspection has been confirmed.\n"
+        " * Cancelled: has been cancelled, can't be confirmed anymore.",
+    )
+
+    vehicle_id = fields.Many2one(
+        "fleet.vehicle",
+        "Vehicle",
+        help="Fleet Vehicle",
+        required=True,
+        states=READONLY_STATES,
+    )
+
+    odometer_id = fields.Many2one(
+        "fleet.vehicle.odometer",
+        "Odometer",
+        help="Odometer measure of the vehicle at the moment of this log",
+    )
+
+    odometer = fields.Float(
+        compute="_get_odometer",
+        inverse="_set_odometer",
```

Comment:
As per [OCA Contributing guidelines](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#37models), computed methods should be named `_compute_field_name` and `_inverse_field_name`

```suggestion
        compute="_compute_odometer",
        inverse="_inverse_odometer",
```

You'll need to change the method as well ;)